src/_P073_7DGT.ino

```diff
@@ -129,7 +129,9 @@ struct P073_data_struct : public PluginTaskData_base {
   void FillBufferWithTemp(long temperature) {
     ClearBuffer();
     char p073_digit[8];
-    sprintf_P(p073_digit, PSTR("%7d"), static_cast<int>(temperature));
+    bool between10and0 = (temperature < 10 && temperature >= 0);      // To have a zero prefix (0.x and -0.x) display between 0.9 and -0.9 degrees,
+    bool between0andMinus10 = (temperature < 0 && temperature > -10); // as all display types use 1 digit for temperatures between 10.0 and -10.0
+    sprintf_P(p073_digit, (between10and0 ? PSTR("     %02d") : (between0andMinus10 ? PSTR("    %03d") : PSTR("%7d"))), static_cast<int>(temperature));
```

Comment:
Why so many spaces?
This plugin does support a variety of displays, which are not limited to 4x 7-segment as far as I know.

Reply:
The smaller displays use the matching right part of the output already (that's existing code), it always formats for the widest supported display (8 digits, 7 used for temperature, 8th for degrees symbol).

Reply:
Check!
Then it should work indeed.

Only thing coming to mind right now....
What if someone has rules to print a string to the display and later print the temp on it.
Previous implementation did not overwrite the current characters on the left of the display, but with this change it does.

Reply:
AFAICS, the display is always filled from left to right, IMHO, that would always overwrite the entire content? ('Spaces' are sent to the display unconditionally, at least to cater for the case of going from 10 to 9 degrees, to not end up with 19...)

Reply:
Thinking about it, I think the only way to put something 'in front' of that temperature is to write it after the temperature is written, and it would need `7dn` or `7dtext` to write it, but I'm not sure it would overwrite the temperature again...
In that case the only way to display 2 separate entities will be formatting it yourself in 1 `7dn` or `7dtext` command.